### PR TITLE
Edit user profiles [WIP]

### PR DESCRIPTION
**Pull request by gsb-eng**
This is continuation to PR 1229 raised by @karan , it's still WIP

WIP

What can be edited:

Name
Password
#423
Future work needed:

Style the page
Allow editing username?
Allow editing email? This will likely be a separate form or page because there can be multiple emails and some can be primary.

### CODE REVIEW

File: warehouse/accounts/views.py [hunk 1]
```diff
@@ -56,6 +57,38 @@ def profile(user, request):
...
+def edit_profile(request, _form_class=forms.EditProfileForm):
+    if not request.user:
+        return HTTPSeeOther(request.route_path("accounts.login"))
+
+    user_service = request.find_service(IUserService, context=None)
+    form = _form_class(request.POST, user_service=user_service)
+
+    if request.method == "POST" and form.validate():
+        username = form.username.data
+        userid = user_service.find_userid(username)
```
Comment (alex) [inline warehouse/accounts/views.py]: This appears to let any user change any other user's data?

Comment (gsb-eng) [inline warehouse/accounts/views.py]: :+1:  , Anyway I just took over this PR, I'll be changing this.

**Comment by brainwane** [comment]
Hi, @gsb-eng! In the Warehouse developer [meeting today](https://wiki.python.org/psf/PackagingWG/2017-12-19-Warehouse) we decided that issue #423 belongs in [the first milestone, the MVP for package maintainers](https://github.com/pypa/warehouse/milestone/8).

So this feature is a priority for us, and we'd love it if you or @karan had time to rebase and update this branch in the next few days!

Thanks and best wishes.

**Comment by di** [comment]
Closing as this is a bit stale, and since we'll be shipping this as part of the [Maintainer MVP milestone](https://github.com/pypa/warehouse/milestone/8).

Thanks for taking a shot at this @gsb-eng!

**Pull request closed without merging**